### PR TITLE
[MIRROR] Spellchecks everything

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/38604
--------------------
:cl: ShizCalev
spellcheck: Did a rather thorough game-wide spellcheck. Please let us know if you find more mistakes!
/:cl:

### CODE REVIEW

**Comment by Carbonhell** [comment]
wtf happened here

**Comment by Carbonhell** [comment]
@yoyobatty ????

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1444,6 +1444,16 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 14							//High cost because of the potential for extreme damage in the hands of a skilled scientist.
 	restricted_roles = list("Research Director", "Scientist")
 
+/datum/uplink_item/role_restricted/blastcannon
```
Comment (Carbonhell) [inline code/modules/uplink/uplink_items.dm:1450]: What is this

**Comment by Carbonhell** [comment]
Cmon, deal with this mirror already, it's been 18 days

**Comment by yoyobatty** [comment]
watch it not work

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**